lib/ex_quiz_it/questions/enums.ex

```diff
@@ -1,7 +1,7 @@
 defmodule ExQuizIt.Questions.Enums do
   @moduledoc false
-  def q_and_a do
-    {:docs_v1, _, :elixir, _, _, _, functions} = Code.fetch_docs(Enum)
+  def q_and_a(module \\ Enum) do
```

Comment:
I wonder if this is a bad idea putting a default argument here. The default is the concern of the LiveView code, not of this module.

Reply:
Good catch! I will put the `Enum` in the mount function instead